src/sinks/influxdb/mod.rs

```diff
@@ -276,7 +276,7 @@ fn encode_string(key: String, output: &mut String) {
     }
 }
 
-fn encode_timestamp(timestamp: Option<DateTime<Utc>>) -> i64 {
+pub fn encode_timestamp(timestamp: Option<DateTime<Utc>>) -> i64 {
```

Comment:
These too should probably be `pub(crate)` or even `pub(in crate::sinks)` (something I just saw was possible in another PR).